.github/workflows/main.yml

```
@@ -28,3 +28,6 @@ jobs:
       run: |
         shopt -s globstar
         pylint **/*.py --disable=fixme
+    - name: Test with pytest
+      run: |
+        pytest
```

```suggestion
      run: pytest
```

Per il resto mi sembra tutto a posto